### DISCUSSION TITLE
novelWriter mention

### DISCUSSION THREAD

**Discussion by johnblommers** [vkbo/novelWriter#757] [General]
I was reading [Writing a Technical Book](https://andregarzia.com/2021/04/writing-a-technical-book.html). About 2/3 of the way down I was pleased to see a reference to novelWriter.

**Comment by vkbo** [comment]
Nice, although novelWriter definitely isn't suited for writing technical books!

I should probably rewrite the blurb on novelWriter, because people seem to be under the impression that it is a proper markdown editor, instead of a plain text editor that uses some markdown syntax.

He also mentions the lack of ebook export, which I haven't decided on yet. the epub format isn't that difficult to generate, and certainly simpler than open document, but there are a lot of tools out there that do this. I have used Calibre in the past, which can create great ebooks from well formatted html. I'm inclined to just provide the well formatted html.
